### DISCUSSION TITLE
Database has been suspended

### DISCUSSION THREAD

**Discussion by MeesEgberts** [vercel/community#7106] [Help]
### Summary

I've downgraded from `pro` to `hobby`. I forgot I had multiple databases in this account. After downgrading I got an email that told me my `free team used 100% of the included free tier usage for Postgres Data Storage (256 MB)`. I've deleted the databases I don't need any more and I'm left with only one database that only uses 44 MB and It still tells me that the database is going to be suspended. 

<img width="1265" alt="Screenshot 2024-06-06 at 09 44 36" src="https://github.com/vercel/community/assets/37342214/4c1987f9-78e6-4a90-a53a-607f491a6384">

### Steps to Reproduce

Create multiple databases with more than 256 MB in total in the `pro` plan and then downgrade to the `hobby` plan. Then delete enough databases so that only one database with less then 256 MB remains. The warning will not go away.

**Comment by pawlean** [comment]
Hi @MeesEgberts!

Thanks for your patience with this. Do you still see this error?

[a comment by an automated account is not shown]
